public/sass/pages/_explore.scss

```diff
@@ -383,6 +383,25 @@
   top: 60px;
   z-index: inherit;
 }
+.ReactTable .filter-links {
```

Comment:
Any chance we can reuse `table-panel-cell-filterable` and `table-panel-filter-link` styles here? They are defined by the dashboard table panel plugin.

Reply:
@davkal, yep, can do. I opted for the new styles alongside the existing `.ReactTable` ones just for cohesiveness with this component definition.

`.table-panel-filter-link` looks like a drop-in replacement, but some additional fidgeting is needed to get the hover and visibility going. That'd mean applying `.table-panel-table` on some parent. Do you intend for `.ReactTable` to eventually reuse everything from `_panel_table.scss`? Please correct me if I misunderstood here.

Reply:
We're still testing react-table and are not completely decided on it. It'd be better to make it work with the grafana styles.